deploy/sre-pruning/110-pruning-images.CronJob.yaml

```diff
@@ -0,0 +1,33 @@
+# Refer to SREP-1104 (and PROD-984)
+# This is on hold until image pruning can be done with a service account
```

Comment:
So I went to SREP-1104, which points to a pretty generic error message on an access.redhat.com page from 2015... and then went to PROD-984, but it's not clear to me why this file is commented out. I've commented on SREP-1104 for some clarification. I can't tell if this is still an issue or not.

Reply:
It's still an issue, unfortunately. For the benefit of folks who don't have access to Jira, [access.redhat.com](https://access.redhat.com/solutions/1553953).

In short: When trying to `oc adm prune images` with a ServiceAccount's authentication token (eg `oc adm prune images --token=/var/run/secrets/kubernetes.io/serviceaccount/token ...`) we get this error: `error: You must use a client config with a token`.

Reply:
I believe `--token` requires a string, not a file. That's the only way I've used it. Have you tried with a string? Or does the file work sometimes?

Reply:
🤦‍♀️ I think you're right! Let me try passing the _contents_ of the token file.

Reply:
`oc sa get-token <serviceaccountname>` can get you that token.

Reply:
I was going to `--token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)` to cut down on the extra serviceaccount permissions.

Reply:
This is the reason I was using a kubeconfig file instead of passing the token string. I don't think you can use a subshell like that in the CronJob spec. I hope I'm wrong about that, because that would be nice to use.

Reply:
This (truncated for clarity) command results still in the error

```
oc adm prune images --token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token) --confirm
F0405 18:38:39.536636      26 helpers.go:116] error: you must use a client config with a token```

Reply:
@dmage @legionus can you weigh in here?

Reply:
Looks like /var/run/secrets/kubernetes.io/serviceaccount/token is empty. Having token is essential for interacting with the registry.

Reply:
@dmage I've verified that `/var/run/secrets/kubernetes.io/serviceaccount/token` is non-empty and that it's the same token as `oc sa get-token sre-pruner-sa`. Additionally, the same token works to run other `oc adm prune` commands.

Reply:
Per https://bugzilla.redhat.com/show_bug.cgi?id=1698004 - Image pruning can be brought into this. This PR will be updated for that.